lib/ansible/plugins/inventory/tower.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: tower
+    plugin_type: inventory
+    authors:
+      - Matthew Jones (@matburt)
+      - Yunfan Zhang (@YunfanZhang42)
+    short_description: Ansible dynamic inventory plugin for Ansible Tower.
+    description:
+        - Imports inventories from Ansible Tower.
```

Comment:
not really an 'import', 'uses tower as inventory source'

Reply:
Will change it, thanks for pointing out.